controllers/monitoring_alert.go

```diff
@@ -501,6 +508,10 @@ func (r *ManagedMCGReconciler) reconcilePrometheus() error {
 			Key: alertRelabelConfigSecretKey,
 		}
 
+		if err := r.configureRHOBSRemoteWrite(); err != nil {
```

Comment:
The reconcilePrometheus method calls this other method that introduces the remote write config in the Prometheus spec